source/includes/fact-connect-sharded.rst

```diff
@@ -0,0 +1,3 @@
+For :ref:`sharded clusters <sharding-sharded-cluster>`, specify the
+hostnames of the :binary:`mongos` instances instead of the
+:binary:`mongod` instances.
```

Comment:
Can we just consolidate this into connection string?

Reply:
also this is not in the Atlas-to-Atlas page

Reply:
See above - the sharding details aren't needed for a replica set

Reply:
-- is sharding exposed in Atlas? I thought those details were hidden. I will  add this include

Reply:
Yes, there are sharded clusters on Atlas. If you are connecting to a sharded cluster on Atlas, then you will need to connect to its connection strings for mongos